proto/cosmos/orm/module/v1alpha1/module.proto

```diff
@@ -1,14 +0,0 @@
-syntax = "proto3";
-
```

Comment:
this is not used anywhere. @aaronc could you confirm that we are good to remove it?

Reply:
we can remove it for now... but it might be used in the future